### PR TITLE
Jj update to latest version

### PR DESCRIPTION
**Pull request by JorMaister**

### CODE REVIEW

**Comment by tibuch** [comment]
Oh sorry! I did not want to change it from `Draft` to `Review requested`. I actually wanted to ask if I should already start the review or wait? 

Was to eager to try out this button :smile:

**Comment by JorMaister** [comment]
No worries, Tim-Oliver! 

I moved it to draft because I found something when I was making some tests. I've written the description in the corresponding g issue of the source repo.

https://github.com/juglab/cryoCARE_pip/issues/29

**Comment by pconesa** [comment]
Insstallation of cryoCARE and test worked like a charm.

File: cryocare/__init__.py [hunk 1]
```diff
@@ -72,21 +71,16 @@ def defineBinaries(cls, env):
...
-        installationCmd += 'pip install mrcfile && '
-        installationCmd += 'pip install csbdeep '
-        # I had the same issue and was able to fix this by setting h5py < 3.0.0.
-        # Looks like here was a 3.0 release of h5py recently where they changed how strings are stored/read.
-        # https://github.com/keras-team/keras/issues/14265
-
-        # Install cryoCARE
-        installationCmd += 'pip install %s==%s &&' % (CRYOCARE, CRYOCARE_DEFAULT_VERSION)
+        # Install cryoCARE and the rest of dependencies
+        installationCmd += 'virtualenv==20.3.1 && '
```
Comment (azazellochg) [inline cryocare/__init__.py:81]: this command does nothing :)

Comment (pconesa) [reply cryocare/__init__.py:81]: true

Comment (JorMaister) [reply cryocare/__init__.py]: You're both right. Fixed! Sorry for the delay, @azazellochg, I missed the notification.

Comment (azazellochg) [reply cryocare/__init__.py]: No problem. I was just installing the plugin for the first time and skimmed through the init code

**Comment by MohamadHarastani** [comment]
Hi @JorMaister can you merge it?

**Comment by JorMaister** [comment]
@pconesa?

**Comment by MohamadHarastani** [comment]
I noticed that the output tomogram name has an extra underscore.. other than that all is good!

**Comment by pconesa** [comment]
Sorry, I'm busy these days. Will find time soon

**Review by pconesa — APPROVED** [review]

**Pull request merged**